spec/factories/factory_bot.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'factory_bot'
+require 'faker'
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
```

Comment:
Do you need all these 5 lines before here? 🤔 Review it, please.

Reply:
removed, this was a desperate try when it wasnt working hahah